recipes/libcurl/all/conanfile.py

```diff
@@ -29,7 +29,8 @@ class LibcurlConan(ConanFile):
                "with_libpsl": [True, False],
                "with_largemaxwritesize": [True, False],
                "with_nghttp2": [True, False],
-               "with_brotli": [True, False]
+               "with_brotli": [True, False],
+               "with_wolfssl": [True, False]
```

Comment:
that might be better then to use something like:
```
"ssl_backend": [None, "openssl", "wolfssl"]
```

Reply:
The changes you suggested will remove option with_openssl. Anyone using option with_openssl will need to rewrite their scripts if doing upgrade. But these are only my observations. Can this be discussed more among experienced conan developers before actually making any changes?

Reply:
@danimtb @uilianries WDYT?

Reply:
I'd prefer to keep the same option and add a new one. So the PR is good for me as it is

Reply:
sure! sorry for the noise @rushikesh90